tests/gpu-tests/run_qwen.sh

```diff
@@ -1,16 +1,17 @@
 # will run all tests starting from only a HF checkpoint. Only requires 1 GPU.
 # also need to define HF_TOKEN for some of the tests
-# model needs to be inside /mnt/datadrive/nemo-skills-test-data/Qwen2.5-Math-7B-Instruct
+# model needs to be inside /mnt/datadrive/nemo-skills-test-data/Qwen3-4B
 # if you need to place it in a different location, modify test-local.yaml config
 # example: HF_TOKEN=<> ./tests/gpu-tests/run.sh
 set -e
 
-export NEMO_SKILLS_TEST_HF_MODEL=/mnt/datadrive/nemo-skills-test-data/Qwen2.5-Math-7B-Instruct
+export NEMO_SKILLS_TEST_HF_MODEL=/mnt/datadrive/nemo-skills-test-data/Qwen3-4B
```

Comment:
this won't be available on the CI machine by default, how about instead we switch to using the `Qwen/Qwen3-4B` here and not a local path? The current code always sets up HF_HOME, so it will only be downloaded one time and will make it simpler to switch models in the future.

Reply:
Good idea! I suggested Qwen3 because it's smaller and 4B should be faster than 7B